components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -211,60 +217,93 @@ function TeamUsage() {
+                                                            {usage.contextURL && toRemoteURL(usage.contextURL)}
+                                                        </span>
+                                                    </div>
+                                                    <div className="flex flex-col my-auto">
+                                                        <span className="text-right text-gray-500 dark:text-gray-400 font-medium">
+                                                            {usage.credits.toFixed(1)}
+                                                        </span>
+                                                        <span className="text-right truncate text-sm text-gray-400 dark:text-gray-500">
+                                                            {getMinutes(usage)}
+                                                        </span>
+                                                    </div>
+                                                    <div className="my-auto" />
+                                                    <div className="flex flex-col col-span-3 my-auto">
+                                                        <span className="text-gray-400 dark:text-gray-500 truncate font-medium">
+                                                            {displayTime(usage.startTime)}
+                                                        </span>
+                                                        <div className="flex">
+                                                            {usage.workspaceType === "prebuild" ? (
+                                                                <UsageIcon className="my-auto w-4 h-4 mr-1" />
+                                                            ) : (
+                                                                ""
+                                                            )}
+                                                            {usage.workspaceType === "prebuild" ? (
+                                                                <span className="text-sm text-gray-400 dark:text-gray-500">Gitpod</span>
+                                                            ) : (
+                                                                <div className="flex">
+                                                                    <img
+                                                                        className="my-auto rounded-full w-4 h-4 inline-block align-text-bottom mr-1 overflow-hidden"
+                                                                        src={usage.user?.avatarURL || ""}
+                                                                        alt="user avatar"
+                                                                    />
+                                                                    <span className="text-sm text-gray-400">
```

Comment:
**nitpick**: I thought I had commented on this one. 🤷 

**thought:** The second row here should be using the same or a dimmed color to convey hierarchy, but also helps users focus on the important information. See also [Miller's law](https://en.wikipedia.org/wiki/Miller's_law) and [Serial-position effect](https://en.wikipedia.org/wiki/Serial-position_effect).

```suggestion
                                                                    <span className="text-sm text-gray-400 dark:text-gray-500">
```

Reply:
Resolving as this has been fixed.